### PR TITLE
Beminfo 1250 Move to enb-bem-techs

### PR DESCRIPTION
**Pull request by tormozz48**
@tavriaforever 
@tadatuta 

Please review this pr.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,9 +45,10 @@
     "bem-site-codestyle": "~0.1.0",
     "borschik-tech-cleancss": "^1.0.3",
     "bower": "~1.4.1",
-    "enb": "0.13.4",
-    "enb-autoprefixer": "0.1.1",
-    "enb-bem-i18n": "^0.1.2",
+    "enb": "~0.15.x",
```
Comment (tadatuta) [inline package.json]: there's no need to use `~` and `.x` at the same time

File: .enb/make.js [hunk 2]
```diff
@@ -4,8 +4,8 @@ module.exports = function(config) {
     config.nodes('*.bundles/*', function(nodeConfig) {
         nodeConfig.addTechs([
             [ require('enb/techs/file-provider'), { target: '?.bemdecl.js' } ],
-            [ require('enb/techs/files') ],
-            [ require('enb/techs/deps') ],
+            [ require('enb-bem-techs').files ],
```
Comment (tadatuta) [inline .enb/make.js]: `enb-bem-techs` exports an object with all required techs so you may use it to reduce ammount of `require` in project code

**Comment by tormozz48** [comment]
@tavriaforever Please review again

File: .enb/make.js [hunk 3]
```diff
@@ -40,8 +36,8 @@ module.exports = function(config) {
 
     config.nodes('desktop.bundles/*', function(nodeConfig) {
         nodeConfig.addTechs([
-            [ require('enb/techs/levels'), { levels: getDesktops(config) } ],
-            [ require('enb-autoprefixer/techs/css-autoprefixer'), {
+            [ techs.bem.levels, { levels: getDesktops(config) } ],
+            [ techs.css.autoprefixer, {
                 browserSupport: [ 'last 2 versions', 'ie 10', 'ff 24', 'opera 12.16' ],
```
Comment (tavriaforever) [inline .enb/make.js]: Давай здесь напишем так `['ie >= 8', 'last 3 versions', '> 2%']`

File: .enb/make.js [hunk 4]
```diff
@@ -62,9 +58,9 @@ module.exports = function(config) {
     config.mode('production', function(modeConfig) {
         config.nodes('*.bundles/*', function(nodeConfig) {
             nodeConfig.addTechs([
-                [ require('enb-borschik/techs/borschik'), { sourceTarget: '?.css', destTarget: '?.min.css', tech: 'cleancss', freeze: true } ],
-                [ require('enb-borschik/techs/borschik'), { sourceTarget: '?.template.i18n.js', destTarget: '?.min.template.i18n.js' } ],
-                [ require('enb-borschik/techs/borschik'), { sourceTarget: '?.js', destTarget: '?.min.js' } ]
+                [ techs.borschik, { sourceTarget: '?.css', destTarget: '?.min.css', tech: 'cleancss', freeze: true } ],
+                [ techs.borschik, { sourceTarget: '?.template.i18n.js', destTarget: '?.min.template.i18n.js' } ],
```
Comment (tavriaforever) [inline .enb/make.js]: `minify: true, freeze: true`

File: .enb/make.js [hunk 5]
```diff
@@ -62,9 +58,9 @@ module.exports = function(config) {
...
+                [ techs.borschik, { sourceTarget: '?.js', destTarget: '?.min.js' } ]
```
Comment (tavriaforever) [inline .enb/make.js]: `minify: true, freeze: true`

**Comment by tavriaforever** [comment]
Код собирается и работает

File: .enb/make.js (hunk 3 above)
Comment (qfox) [inline .enb/make.js]: А откуда инфа про `>2%`

**Comment by tormozz48** [comment]
@tavriaforever Please review it again

**Comment by tavriaforever** [comment]
ok

**Pull request merged**